dexterity/membrane/profiles/uninstall/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<metadata>
+  <version>1001</version>
+  <dependencies>
+        <dependency>profile-plone.app.referenceablebehavior:default</dependency>
+        <dependency>profile-plone.app.registry:default</dependency>
```

Comment:
metadata.xml is not strictly needed in an uninstall profile, though having a version in it is fine. I don't remember if I usually add the file or not. :-)
But the dependencies list should be empty, otherwise GenericSetup will install these dependencies. I think you don't want that here.

Reply:
removed dependencies